google/cloud/storage/examples/storage_bucket_samples.cc

```diff
@@ -249,15 +310,22 @@ void GetBucketDefaultKmsKey(google::cloud::storage::Client client, int& argc,
   auto bucket_name = ConsumeArg(argc, argv);
   //! [get bucket default kms key] [START storage_get_bucket_default_kms_key]
   namespace gcs = google::cloud::storage;
+  using google::cloud::StatusOr;
   [](gcs::Client client, std::string bucket_name) {
-    gcs::BucketMetadata metadata = client.GetBucketMetadata(bucket_name);
-    if (not metadata.has_encryption()) {
-      std::cout << "The bucket " << bucket_name
+    StatusOr<gcs::BucketMetadata> metadata =
```

Comment:
Samples are inconsistent with their usage of meta and metadata. s/metadata/meta/ for consistency. I'd prefer `metadata`, but prior PR's have used `meta` so that's fine with me.